source/Calamari.Testing/TestCalamariCommandResult.cs

```diff
@@ -9,18 +9,20 @@ public class TestCalamariCommandResult
     {
         public TestCalamariCommandResult(int exitCode, IReadOnlyDictionary<string, TestOutputVariable> outputVariables,
             IReadOnlyList<TestScriptOutputAction> outputActions, IReadOnlyList<ServiceMessage> serviceMessages,
-            string? resultMessage, IReadOnlyList<CollectedArtifact> artifacts, string fullLog)
+            string? resultMessage, IReadOnlyList<CollectedArtifact> artifacts, string fullLog, string workingPath)
         {
             ExitCode = exitCode;
             OutputVariables = outputVariables;
             OutputActions = outputActions;
             ServiceMessages = serviceMessages;
             ResultMessage = resultMessage;
             FullLog = fullLog;
+            WorkingPath = workingPath;
             Artifacts = artifacts;
         }
 
         public string FullLog { get; }
+        public string WorkingPath { get; }
```

Comment:
Where is this used?

Reply:
I think originally i used, but then i changed things. That said it is useful when using real packages is we need to assert files on disk